.github/workflows/posix.yml

```diff
@@ -61,7 +61,7 @@ jobs:
             env_file: actions-310-numpydev.yaml
             pattern: "not slow and not network and not single_cpu"
             pandas_testing_mode: "deprecate"
-            test_args: "-W error"
+            test_args: "-W error::DeprecationWarning:numpy"
```

Comment:
will this still catch FutureWarnings that we issue?

Reply:
Nope, only the DeprecationWarnings from numpy. Do we want to fail the build on our FutureWarnings?

Reply:
its possible we want to (but not this PR)